hera_cal/tests/test_smooth_cal.py

```diff
@@ -42,27 +42,31 @@ def test_dpss_filters(self):
         freq_scale = 0.5 / (np.diff(freqs)[0] / 1e6)
         time_scale = 0.5 / (np.diff(times)[0] * 3600 * 24)
         # Show that dpss_filter generates filters for a 2D grid of times and freqs
-        design_matrix = smooth_cal.dpss_filters(
+        time_filters, freq_filters = smooth_cal.dpss_filters(
             times=times, freqs=freqs, time_scale=1.01 / time_scale, freq_scale=1.01 / freq_scale
         )
-        assert design_matrix.shape[0] == int(freqs.shape[0] * times.shape[0])
+        assert time_filters.shape[0] == times.shape[0]
+        assert freq_filters.shape[0] == freqs.shape[0]
         # test that error is thrown when filtering scale is too fine
         assert pytest.raises(ValueError, smooth_cal.dpss_filters, times=times, freqs=freqs,
                              time_scale=0.99 / time_scale, freq_scale=1.01 / freq_scale)
         assert pytest.raises(ValueError, smooth_cal.dpss_filters, times=times, freqs=freqs,
                              time_scale=1.01 / time_scale, freq_scale=0.99 / freq_scale)
+        # test approximate estimator of the eigenvalues
```

Comment:
Should probably add some tests/asserts on `time_filters` and `freq_filters` like those above

Reply:
I removed this test because I removed the `Nmax` keyword argument from this function, but added some additional tests